templates/controllers/wizard/fileUpload/form/submissionFileMetadataForm.tpl

```diff
@@ -30,6 +30,25 @@
 		{fbvFormSection title="submission.upload.noteToAccompanyFile"}
 			{fbvElement type="textarea" id="note" height=$fbvStyles.height.SHORT}
 		{/fbvFormSection}
+		{if $submissionSettingsRevisions}
+			<label for="submissionSettingsRevision">{translate key="submission.upload.assignMetadata"}:</label>
+			<select name="submissionSettingsRevision">
```

Comment:
Suggest writing this with `{fbvElement type="select" ...}` rather than a custom `<select>` element. Not that we're totally happy with this approach but it is consistent with the rest of the codebase.